### PR TITLE
SNMP Version Usage

### PR DESCRIPTION
**Pull request by drewjj**
This reworks the current SNMPwalk and SNMPset features to decide which SNMP commands to use based on a target RSU's SNMP version in PostgreSQL.
- SNMP relies on the snmp_versions table
- Added unit tests and reworked others. All SNMP tests relating to Kapsch removed since we can no longer guarantee functionality on any latest Kapsch firmware (CDOT does not have a support agreement with them).
- SQL database changes for default values of the snmp_versions table. The previous values didn't make a whole lot of sense to me.

### CODE REVIEW

File: services/api/src/rsufwdsnmpset.py [hunk 1]
```diff
@@ -331,5 +333,5 @@ def delete(request):
   if errors:
     return f"The provided args does not match required values: {str(errors)}", 400
   
-  response, code = config_del(request['rsu_ip'], request['manufacturer'], request['snmp_creds'], request['args']['msg_type'], request['args']['rsu_index'])
+  response, code = config_del(request['rsu_ip'], request['snmp_version'], request['snmp_creds'], request['args']['msg_type'], request['args']['rsu_index'])
```
Comment (jacob6838) [inline services/api/src/rsufwdsnmpset.py]: We could start using named arguments for these longer methods. Makes them more easily readable

**Review by jacob6838 — COMMENTED** [review]
Looks good to me. In hindsight, having a version code and a nickname was pretty unnecessary for SNMP Version, but it works.

File: services/api/src/rsufwdsnmpset.py (hunk 1 above)
Comment (drewjj) [reply services/api/src/rsufwdsnmpset.py]: This has been addressed to the way you suggested.

**Comment by jacob6838** [comment]
I am having issues with the fabric library for tests, going to fiddle with that a little then confirm tests run!

**Comment by jacob6838** [comment]
I keep getting this error:
ImportError: DLL load failed while importing _bcrypt: The specified procedure could not be found.

image

**Review by jacob6838 — APPROVED** [review]
tests work with Python 3.12!

**Review by mwodahl — APPROVED** [review]
This looks good to me & unit tests are passing!

**Pull request merged**